### PR TITLE
PyCon Ar 2011

### PR DESCRIPTION
**Pull request by akielbowicz**
title: PyCon Ar 2011
dir: pycon-ar-2011
youtube_list:
    https://www.youtube.com/playlist?list=PL_9_IFqoW6HxBL-2srkF1dDONhn6GDElN
language: spa
dates:
    begin: 2011-09-23
    end: 2011-09-23

### CODE REVIEW

File: pycon-ar-2011/videos/pyconar-2011-alejandro-cura-alecu-todo-sobre-deferreds-pyar.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "description": "\"Los Diferidos\" es posiblemente el tema m\u00e1s complejo que tiene que entender un reci\u00e9n llegado al framework Twisted. Pero los Deferreds son una estructura de control muy poderosa, que permite organizar el c\u00f3digo asincr\u00f3nico de una manera muy flexible y f\u00e1cil de testear.  La conferencia tuvo lugar en la ciudad de Jun\u00edn ( Buenos Aires, Argentina ) en Septiembre de 2011, y fu\u00e9 organizada por PyAr ( Python Argentina ).",
+  "language": "spa",
+  "recorded": "2011-09-23",
+  "speakers": [
+    "Alejandro Cura ( alecu )"
```
Comment (redapple) [inline pycon-ar-2011/videos/pyconar-2011-alejandro-cura-alecu-todo-sobre-deferreds-pyar.json]: You can probably drop the `(alecu)`

File: pycon-ar-2011/videos/pyconar-2011-charlas-relampago-dia-1-pyar.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "description": "Las charlas rel\u00e1mpago del primer d\u00eda de PyConAr 2011. Transcurrieron en una linda tarde de Septiembre de 2011 en Jun\u00edn, Buenos Aires, Argentina. En el evento de Python m\u00e1s importante de Argentina, organizado por PyAr.",
+  "language": "spa",
+  "recorded": "2011-09-23",
+  "speakers": [
+    ""
```
Comment (redapple) [inline pycon-ar-2011/videos/pyconar-2011-charlas-relampago-dia-1-pyar.json]: For lightning talks, we usually use `"Various Speakers"`

File: pycon-ar-2011/videos/pyconar-2011-wesley-chun-what-is-google-app-engine-pyar.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  "language": "eng",
+  "recorded": "2011-09-23",
+  "speakers": [
+    "Wesley Chun"
+  ],
+  "thumbnail_url": "https://i.ytimg.com/vi/xkRYG5bMXjs/hqdefault.jpg",
+  "title": "What is Google App Engine? PYAR",
```
Comment (redapple) [inline pycon-ar-2011/videos/pyconar-2011-wesley-chun-what-is-google-app-engine-pyar.json]: "PYAR" at the end can be removed I think

File: pycon-ar-2011/videos/pyconar-2011-hector-sanchez-gonzalo-odiardmanuel-quinones-entorno-educativo-sugar-pyar.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
+{
+  "description": "Descripci\u00f3n de la charla: Mostraremos el entorno educativo desarrollado en python, que viene en las computadoras de OLPC. Ense\u00f1aremos como desarrallorar actividades educativa en python para esta plataforma Tambi\u00e9n como poder participar y sumarse a este proyecto, como programador Python.\nLa conferencia tuvo lugar en la ciudad de Jun\u00edn ( Buenos Aires, Argentina ) en Septiembre de 2011, y fu\u00e9 organizada por PyAr ( Python Argentina ).",
+  "language": "spa",
+  "recorded": "2011-09-23",
+  "speakers": [
+    "H\u00e9ctor Sanchez Gonzalo Odiard",
+    "Manuel Qui\u00f1ones"
+  ],
+  "thumbnail_url": "https://i.ytimg.com/vi/CBcmeHhYBdA/hqdefault.jpg",
+  "title": "Entorno educativo Sugar PYAR",
```
Comment (redapple) [inline pycon-ar-2011/videos/pyconar-2011-hector-sanchez-gonzalo-odiardmanuel-quinones-entorno-educativo-sugar-pyar.json]: "PYAR" here in the title is probably not necessary

**Comment by akielbowicz** [comment]
@redapple made the requested changes. 
Thanks for the review.

**Pull request merged**